apiserver/params/applications.go

```diff
@@ -314,6 +314,15 @@ type DestroyConsumedApplicationsParams struct {
 // RemoteApplication.Destroy call.
 type DestroyConsumedApplicationParams struct {
 	ApplicationTag string `json:"application-tag"`
+
+	// Force controls whether or not the destruction process ignores
+	// operational errors. When true, the process will ignore them.
+	Force bool `json:"force"`
```

Comment:
I think this should be *bool with omitempty in json tag... to cater for backward compatibility.

Reply:
Done

Reply:
Done